octokit/client.py

```diff
@@ -9,11 +9,13 @@
 
 # https://code.google.com/p/uri-templates/wiki/Implementations
 
-from .resources import Resource
+from resources import Resource
```

Comment:
@tchangg and if you look at the tests for your push (https://travis-ci.org/octokit/octokit.py/builds/89161152) you can see that with python3+, you actually need the relative import for it to work.